tritium-core/src/main/java/com/palantir/tritium/event/CompositeInvocationEventHandler.java

```diff
@@ -120,12 +119,10 @@ public String toString() {
 
     private static void preInvocationFailed(InvocationEventHandler<? extends InvocationContext> handler,
             Object instance, Method method, Object[] args, Exception exception) {
-        logger.warn("Exception handling preInvocation({}): "
-                        + "invocation of {}.{} with arguments {} on {} threw: {}",
+        logger.warn("Exception handling preInvocation({}): invocation of {}.{} on {} threw: {}",
                 UnsafeArg.of("handler", handler),
                 SafeArg.of("class", method.getDeclaringClass().getCanonicalName()),
                 SafeArg.of("method", method.getName()),
-                UnsafeArg.of("args", Arrays.toString(args)),
                 UnsafeArg.of("instance", instance),
                 UnsafeArg.of("exception", exception),
```

Comment:
unrelated to this change, but this line can be removed, and we can log `"Exception handling preInvocation({}): invocation of {}.{} on {} threw"`, the stack trace already provides more information.